include/sound/soc-dpcm.h

```diff
@@ -61,6 +61,22 @@ enum snd_soc_dpcm_trigger {
 	SND_SOC_DPCM_TRIGGER_BESPOKE,
 };
 
+/**
```

Comment:
you have explained why in the commit message, but can you add a what this commit does to DPCM updates (as first paragraph).

Reply:
I would say
```
This is a preparation for the host part of a virtualised VirtIO audio
host-guest driver pair. It adds a "mode" parameter to
soc_dpcm_runtime_update() to allow it to be used when stopping
streaming in a virtual machine, which requires forced DPCM audio path
pruning.

Usually we update the full DAPM path during PCM stream operations, however
when the DAPM path is split across multiples machines (as it would be in a virtualised
environment) we have to take care and only update DAPM DPCM paths as follows

1) NEW paths at stream start because X.....

2) OLD paths at stream stop because Y....

There is no impact for regular non virtualised streams.
```